kotsadm/pkg/app/app.go

```diff
@@ -42,27 +52,71 @@ func Get(id string) (*App, error) {
 		zap.String("id", id))
 
 	db := persistence.MustGetPGSession()
-	query := `select id, slug, name, current_sequence, is_airgap, restore_in_progress_name, update_checker_spec from app where id = $1`
+	query := `select id, name, license, upstream_uri, icon_uri, created_at, updated_at, slug, current_sequence, last_update_check_at, is_airgap, snapshot_ttl_new, snapshot_schedule, restore_in_progress_name, restore_undeploy_status, update_checker_spec from app where id = $1`
 	row := db.QueryRow(query, id)
 
 	app := App{}
 
+	var licenseStr sql.NullString
+	var upstreamURI sql.NullString
+	var iconURI sql.NullString
+	var updatedAt sql.NullTime
 	var currentSequence sql.NullInt64
+	var lastUpdateCheckAt sql.NullString
+	var snapshotTTLNew sql.NullString
+	var snapshotSchedule sql.NullString
 	var restoreInProgressName sql.NullString
+	var restoreUndeployStatus sql.NullString
 	var updateCheckerSpec sql.NullString
 
-	if err := row.Scan(&app.ID, &app.Slug, &app.Name, &currentSequence, &app.IsAirgap, &restoreInProgressName, &updateCheckerSpec); err != nil {
+	if err := row.Scan(&app.ID, &app.Name, &licenseStr, &upstreamURI, &iconURI, &app.CreatedAt, &updatedAt, &app.Slug, &currentSequence, &lastUpdateCheckAt, &app.IsAirgap, &snapshotTTLNew, &snapshotSchedule, &restoreInProgressName, &restoreUndeployStatus, &updateCheckerSpec); err != nil {
 		return nil, errors.Wrap(err, "failed to scan app")
 	}
 
+	app.License = licenseStr.String
+	app.UpstreamURI = upstreamURI.String
+	app.IconURI = iconURI.String
+	app.LastUpdateCheckAt = lastUpdateCheckAt.String
+	app.SnapshotTTL = snapshotTTLNew.String
+	app.SnapshotSchedule = snapshotSchedule.String
+	app.RestoreInProgressName = restoreInProgressName.String
+	app.RestoreUndeployStatus = restoreUndeployStatus.String
+	app.UpdateCheckerSpec = updateCheckerSpec.String
+
+	if updatedAt.Valid {
+		app.UpdatedAt = &updatedAt.Time
+	}
+
 	if currentSequence.Valid {
 		app.CurrentSequence = currentSequence.Int64
 	} else {
 		app.CurrentSequence = -1
 	}
 
-	app.RestoreInProgressName = restoreInProgressName.String
-	app.UpdateCheckerSpec = updateCheckerSpec.String
+	if app.CurrentSequence != -1 {
+		query = `select preflight_spec, config_spec from app_version where app_id = $1 AND sequence = $2`
+		row = db.QueryRow(query, id, app.CurrentSequence)
+
+		var preflightSpec sql.NullString
+		var configSpec sql.NullString
+
+		if err := row.Scan(&preflightSpec, &configSpec); err != nil {
+			return nil, errors.Wrap(err, "failed to scan app_version")
+		}
+
+		if preflightSpec.Valid && preflightSpec.String != "" {
+			app.HasPreflight = true
+		}
+		if configSpec.Valid && configSpec.String != "" {
+			app.IsConfigurable = true
+		}
+	}
+
+	bundleCommand := fmt.Sprintf(`
+	curl https://krew.sh/support-bundle | bash
```

Comment:
this will have spaces in the front when rendered.  you can just use `\n` and regular quotes